bin/check-postgres-replication.rb

```diff
@@ -130,7 +136,11 @@ def run
                             sslmode: ssl_mode,
                             connect_timeout: config[:timeout])
 
-    slave = conn_slave.exec('SELECT pg_last_xlog_receive_location()').getvalue(0, 0)
+    slave = if Gem::Version.new(pg_vsn) > Gem::Version.new('9.6')
```

Comment:
same thing, as we are checking the same thing twice it might make sense to make the condition a function and pass in the version and client/master.

Reply:
Makes sense.